### PR TITLE
Update to 1.1.5 image

### PR DESCRIPTION
**Pull request by cdrage**
**What type of PR is this?**
> Uncomment only one ` /kind` line, and delete the rest.
> For example, `> /kind bug` would simply become: `/kind bug`

/kind feature

**What does does this PR do / why we need it**:

Updates to 1.1.5 odo-init-image.

**Which issue(s) this PR fixes**:

N/A

**PR acceptance criteria**:

N/A

**How to test changes / Special notes to the reviewer**:

N/A

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

File: pkg/devfile/adapters/common/utils.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ const (
 
 	// Default Image that will be used containing the supervisord binary and assembly scripts
 	// use GetBootstrapperImage() function instead of this variable
-	defaultBootstrapperImage = "registry.access.redhat.com/openshiftdo/odo-init-image-rhel7:1.1.3"
+	defaultBootstrapperImage = "registry.redhat.io/ocp-tools-4/odo-init-container-rhel8:1.1.5"
```
Comment (kadel) [inline pkg/devfile/adapters/common/utils.go:35]: This doesn't look like. 
`registry.redhat.io` is a registry that requires authentication.

Comment (cdrage) [reply pkg/devfile/adapters/common/utils.go:35]: Woops!

Comment (mohammedzee1000) [reply pkg/devfile/adapters/common/utils.go:35]: Can you update registry to registry.access.redhat.com

**Pull request merged**